adapters/Vungle/VungleAdapter/GADMAdapterVungleDelegate.h

```diff
@@ -14,10 +14,13 @@
 
 /// Vungle adapter ad type.
 typedef NS_ENUM(NSUInteger, GADMAdapterVungleAdType) {
-  GADMAdapterVungleAdTypeUnknown,       ///< Unknown adapter type.
-  GADMAdapterVungleAdTypeRewarded,      ///< Rewarded adapter type.
-  GADMAdapterVungleAdTypeInterstitial,  ///< Interstitial adapter type.
-  GADMAdapterVungleAdTypeBanner         ///< Banner adapter type.
+  GADMAdapterVungleAdTypeUnknown,            ///< Unknown adapter type.
+  GADMAdapterVungleAdTypeRewarded,           ///< Rewarded adapter type.
+  GADMAdapterVungleAdTypeInterstitial,       ///< Interstitial adapter type.
+  GADMAdapterVungleAdTypeMREC,               ///< MREC adapter type.
+  GADMAdapterVungleAdTypeBanner,             ///< Banner adapter type.
+  GADMAdapterVungleAdTypeShortBanner,        ///< ShortBanner adapter type.
+  GADMAdapterVungleAdTypeLeaderboardBanner   ///< LeaderboardBanner adapter type.
```

Comment:
Are these needed? Is it necessary to have 4 types for Banner ads? Can we stick to one Banner `GADMAdapterVungleAdType`?

Reply:
@joshuajplagonera Hi joshuajplagonera, all of these types are needed. Because MREC ads and Banner ads are using different APIs to load ads. When loading Banner ads, we also need to pass the Banner size type (Banner[320 * 50], short Banner[300 * 50] or Leaderboard Banner[728*90]) to Vungle SDK,  so we need different Banner types to distinguish Banner ads of different sizes. Thanks a lot. 🙂